data/Language/VisualBasic.yaml

```diff
@@ -0,0 +1,148 @@
+identifier: Visual Basic
+full_name: Visual Basic
+wikidata: Q2378
+grammar_file: https://github.com/antlr/grammars-v4/blob/master/vb6/VisualBasic6.g4
+aliases:
+  - vb
+extensions:
+  - vb
+  - bas
+line_continuation: '_'
+delimiters:
+  # comment delimiters
+  - single_quote
```

Comment:
I check online and saw that comments are written by putting a single quote on the starting of the line. So this might now be applicable here.

Reply:
@Man-Jain I don't think its necessary that a comment starts on a newline. I saw a few codes where they were used at the end of the line too.

Reply:
You can put the quote at the end of the line and it still will be a comment. But that isn't the correct way and we have to do things the correct way. 
In python also we can use hash at both start and end of the line. This doesn't mean we can put that as comment delimiter in python's file cause the correct way is to use it once.

Reply:
Oh, I understood my mistake, that delimiter is for strings, I'll have to define a new delimiter.

Reply:
Added a new delimiter `single_quote_comment` and appended that to the list.